### ISSUE TITLE
Extension breaks on annotations

### ISSUE THREAD

**Issue by jonathanwiesel** [btamburrino/VSCode-Apex-Javadoc#9]
I've noticed that if a method contains an annotation the comment stub fails to create at seems to break the running extension since it stops generating stubs unless VSCode is restarted.

For example

```apex
global class LightningLoginFormController {

    // try adding your comment here
    @AuraEnabled
    public static String login(String username, String password, String startUrl) {
        try{
            ApexPages.PageReference lgn = Site.login(username, password, startUrl);
            aura.redirect(lgn);
            return null;
        }
        catch (Exception ex) {
            return ex.getMessage();            
        }
    }
}
```

**Comment by btamburrino** [comment]
@jonathanwiesel Just pushed 1.2.1 to the VSCode Marketplace that fixed this as well as #10. Embarrassing oversights on my part. Thank you so much for reporting them!!

**Comment by jonathanwiesel** [comment]
Nothing to be ashamed of here, we've all been in this kind of situations 😉

I'm grateful for your speedy response time
